modules/example/src/smithy4s/example/product/package.scala

```diff
@@ -0,0 +1,8 @@
+package smithy4s.example
+
+package object product {
+  type ObjectService[F[_]] = smithy4s.kinds.FunctorAlgebra[ObjectServiceGen, F]
+  val ObjectService = ObjectServiceGen
+
+
```

Comment:
thought: if you can remove these extra newlines that'd be pretty cool

Reply:
but I don't want to extend this PR any more so let's leave it for future generations :P

Reply:
this seems to come from past code because it seems other package files have the same issue. BTW I think recent changes have removed/moved the example module so I think these files shouldn't exist, will remove them

Reply:
removed at [7cab3a3](https://github.com/disneystreaming/smithy4s/pull/1011/commits/7cab3a3edc6703c16ce69fc840634a4e009f6060)